loops/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace loops
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // TaskOne();
+            // TaskTwo();
+            TaskThree(); 
+        }
+
+        static void TaskOne(){
+            Console.WriteLine("Pls input n: ");
+            int n = Int32.Parse(Console.ReadLine());
+            double result = 0;
+
+            for(double i = 1; i < n + 1; i++){
+                result += (i + 1) / i;
+                // Console.WriteLine(result);
+            }

```

Comment:
Мусор (закомментированный код) лучше убирать, а если в теле цикла один оператор, то `{ }` можно не писать.
```suggestion
            for(double i = 1; i < n + 1; i++)
                result += (i + 1) / i;

```